configure

```diff
@@ -0,0 +1,58 @@
+#!/bin/sh
+
+AUTO_QMAIL=/var/qmail
+
+for arg in $*; do
+	case "${arg}" in
+	--prefix=*)
+		AUTO_QMAIL=$(echo "${arg}" | sed 's#^--prefix=##')
```

Comment:
is AUTO_QMAIL=${arg#*=} to avoid instead of the more explicit AUTO_QMAIL=$(echo "${arg}" | sed 's#^--prefix=##')? This is a minor detail, I am fine with both styles.

Reply:
That is a bash extension.

Reply:
It does not look like so, but the sed version is also very clear and good.

> `${parameter#[word]}`
>   Remove Smallest Prefix Pattern. The word shall be expanded to produce a pattern. The parameter expansion shall then result in parameter, with the smallest portion of the prefix matched by the pattern deleted. If present, word shall not begin with an unquoted '#'.
> https://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html#tag_18_06_02